package.json

```diff
@@ -51,7 +51,6 @@
     "@types/node-fetch": "^2.6.6",
     "@types/node-forge": "^1.3.7",
     "@typescript-eslint/eslint-plugin": "^6.6.0",
-    "discord.js": "^14.15.3",
```

Comment:
This can't be here or it will prune discord.js during the image creation XD

Reply:
weird- idk how discordjs got added as a dev dep

Reply:
Me, I added it by accident 😂